### DISCUSSION TITLE
Release 9 - July 2024

### DISCUSSION THREAD

**Discussion by sinecurelife** [PaulJerimy/SecCertRoadmapHTML#126] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/PaulJerimy/SecCertRoadmapHTML/releases/tag/v9'>Release 9 - July 2024</a>.</em>

**Comment by peterverwolf** [comment]
_From changelog:
Shortened Security+, SSCP, GSEC, Programming Languages, CASP+, CISSP, CISSP Concentrations, and GSE to cover 4 spaces in only GRC to make room for more certifications_

Why are these certifications are less broad positioned then before? CISSP is now only in the GRC domain, but I think the position in the previous version was more correctly. 

Aside from this, nice to see you are working on this again. Thanks for you work!

**Comment by edsecru** [comment]
Hello @sinecurelife Paul, thank you for all of your hard work in creating this map. This has been game changing for my career and advancement of understanding.

I was wondering if you could add the CMMC CCA Certification? I see you already have the CMMC CCP on there which is the certification below the CCA. I would expect the CCA to go between Intermediate/Expert categories in the Security and Risk Management domain.

**Comment by kyrioskova** [comment]
Just update

For Comptia CASP+, now it being rename as SecurityX.
CASP+ Name Change To SecurityX
CompTIA Advanced Security Practitioner (CASP+) will be re-branded to [SecurityX](https://www.comptia.org/certifications/securityx) with the release of the next exam version CAS-005 on December 17, 2024.
[https://www.comptia.org/certifications/securityx]

**Comment by hajdus** [comment]
hi @sinecurelife 
do you plan add PJPT Cert from tcm sec to map?
https://certifications.tcm-sec.com/pjpt/

**Comment by ludou-art** [comment]
@sinecurelife 
Can you give us a timeline for your next update?
Recently,a web page has even popped up that imitates your great work.
